content-src/components/Spotlight/Spotlight.js

```diff
@@ -8,9 +8,9 @@ const SpotlightItem = React.createClass({
     const imageUrl = site.images[0].url;
     const description = site.description;
     return (<li className="spotlight-item">
-      <div className="spotlight-image" style={{backgroundImage: `url(${imageUrl})`}} ref="image">
+      <a href={site.url} className="spotlight-image" style={{backgroundImage: `url(${imageUrl})`}} ref="image">
         <SiteIcon className="spotlight-icon" site={site} ref="icon" height={32} width={32} />
-      </div>
+      </a>
```

Comment:
FWIW, this may cause a mid-air collision w/ @k88hudson's [#126](https://github.com/mozilla/activity-streams/pull/126/files#diff-b0472e345878c3704eb056b33862e4d4) PR.